app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -180,38 +181,65 @@ public void onConnected(Bundle bundle) {
         public Person resolved;
     }
 
-    public class OrganizerAdapter extends ArrayAdapter<Organizer> {
+    public class OrganizerAdapter extends RecyclerView.Adapter<OrganizerAdapter.ViewHolder> {
 
-        private LayoutInflater inflater;
+        // SparseArray is recommended by most Android Developer Advocates
+        // as it has reduced memory usage
+        private SparseArrayCompat<Organizer> organizers;
 
-        public OrganizerAdapter(Context context, int resource) {
-            super(context, resource);
-
-            inflater = LayoutInflater.from(context);
+        public OrganizerAdapter() {
+            organizers = new SparseArrayCompat<>();
         }
 
         @Override
-        public View getView(final int position, View convertView, ViewGroup parent) {
-            if (convertView == null) {
-                convertView = inflater.inflate(R.layout.list_tagged_organizer_item, parent, false);
-            }
+        public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+            View v =  LayoutInflater.from(parent.getContext()).inflate(R.layout.list_tagged_organizer_item, parent, false);
+            return new ViewHolder(v);
+        }
 
-            Organizer o = getItem(position);
+        @Override
+        public void onBindViewHolder(ViewHolder holder, int position) {
+            final Organizer o = organizers.get(position);
+
+            Picasso.with(holder.itemView.getContext())
```

Comment:
We have a Picasso instance in App class. Is this already using it? Or should we explicitly use that instance.